templates/verdaccio.service.j2

```diff
@@ -5,7 +5,7 @@ Description=verdaccio Service
 Type=simple
 User=verdaccio
 WorkingDirectory={{ verdaccio_datadir }}
-ExecStart=/usr/local/bin/verdaccio -c {{ verdaccio_configdir }}/verdaccio.yml
+ExecStart={{ npm_config_prefix }}/bin/verdaccio -c {{ verdaccio_configdir }}/verdaccio.yml
```

Comment:
What does the command return on your machine? I wonder it will work everywhere, as it returned: `/usr/local/lib/npm` on my machine.

Reply:
For us, it returns: `/usr/local`

Reply:
in npm cli package, it uses the update-notifier package to remind user to update their package. However, the update-notifier is running as sub-process, which will give the notification every other interval setting(1 day in default). 
We can add extra args to suppress that behaviour. Sorry for late reply.
npm config get prefix --no-update-notifier @030